content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -168,19 +138,31 @@ to manage the lifecycle of Istio.
 
 1. Install the `istio` chart:
 
-    {{< text bash >}}
+    * For a production environment use the recommended Helm values or customize as needed:
 
+    {{< text bash >}}
     $ helm install istio --name istio --namespace istio-system
+    {{< /text >}}
+
+    * To try the `bookinfo` demo, use Helm values customized for the `bookinfo` application:
 
+    {{< text bash >}}
+    $ helm install istio --name istio --values install/kubernetes/helm/istio/values-istio-demo.yaml --namespace istio-system
     {{< /text >}}
 
-1. Uninstall steps:
+## Uninstall
+
+* If you installed using Option 1 with `helm template`, uninstall with this command:
 
     {{< text bash >}}
+    $ kubectl delete namespace istio-system
```

Comment:
this is insufficient.  while deleting hte istio-system namesapce will get rid of most istio stuff, there are some things installed in the default namespace that must also be removed.

Reply:
I merely re-organized the uninstall steps do they are not right after the install.

What else is needed and can it be done in a separate PR? This is meant to unblock people who follow these instructions for testing.